python/GangaDirac/Lib/Backends/DiracBase.py

```diff
@@ -495,20 +497,25 @@ def _getStateTime(job, status):
             if not job.subjobs and status in b_list:
                 for childstatus in b_list:
                     if job.backend.id:
-                        logger.debug("Accessing getStateTime() in diracAPI")
-                        dirac_cmd = "getStateTime(%d,\'%s\')" % (job.backend.id, childstatus)
-                        be_statetime = execute(dirac_cmd)
+               #         logger.debug("Accessing getStateTime() in diracAPI")
+                        #be_statetime = execute("getStateTime(%d,\'%s\')" % (job.backend.id, childstatus))
```

Comment:
The general shape is I think I'll be pushing for merge at the end of today, aside from playing with the logger here there isn't anything else I'm too worried about, (although I'd like to retest based upon some changes elsewhere in the PR)